### PR TITLE
Add kafka ngrok example

### PR DESCRIPTION
**Pull request by rmoff**

### CODE REVIEW

File: kafka-ngrok/README.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+----
+
+Once up, find out your Kafka broker host/post that is available on the internet:
+
+[source,bash]
+----
+curl -s localhost:4040/api/tunnels | jq -r '.tunnels[0].public_url' | sed 's/tcp:\/\///g'
+----
+
+WARNING: There is **NO SECURITY** defined on this Kafka cluster. Anyone scanning the ngrok address/port may find this and be able to access your data.
```
Comment (gunnarmorling) [inline kafka-ngrok/README.adoc:28]: Good to call this out. Alternatively, I have a set-up for basic auth [here](https://github.com/kcctl/kcctl/blob/main/docker-compose.broker-auth.yml). Maybe worth adopting. Still not production-grade or anything, but it's at least not leaving the house completely unlocked.

File: kafka-ngrok/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+  zookeeper:
+    image: confluentinc/cp-zookeeper:7.5.1
+    container_name: zookeeper
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
+      ZOOKEEPER_TICK_TIME: 2000
+
+  broker:
+    image: confluentinc/cp-kafka:7.5.1
```
Comment (gunnarmorling) [inline kafka-ngrok/docker-compose.yml:24]: How about using the new official Apache image instead? And maybe no ZK?

**Review by gunnarmorling — COMMENTED** [review]
Nice one. Some minor comments inline. Taking a step back, we probably should look into re-organizing the examples, into more foundational things like this, vs. Decodable-specific stuff, etc.

File: kafka-ngrok/docker-compose.yml (hunk 2 above)
Comment (rmoff) [reply kafka-ngrok/docker-compose.yml:24]: sure. next time :) this is just to get it out the door.

File: kafka-ngrok/README.adoc (hunk 1 above)
Comment (rmoff) [reply kafka-ngrok/README.adoc:28]: does that setup work with the Confluent images?

Comment (gunnarmorling) [reply kafka-ngrok/README.adoc:28]: I don't know, never used those ;)

Comment (rmoff) [reply kafka-ngrok/README.adoc:28]: kk. for now I just wanna get this into the repo. maybe on the boston flight I will improve it :)

**Review by gunnarmorling — APPROVED** [review]
LGTM, with some room for potential future improvements ;)

**Comment by rmoff** [comment]
canadian-thank-you-gif-by-netflix

**Pull request merged**